talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -96,6 +99,29 @@ void PeerConnectionDependencyFactory::
       GlobalConfiguration::GetAEC3Enabled()) {
     field_trial_ += "OWT-EchoCanceller3/Enabled/";
   }
+  // Handle port ranges.
+  int audio_min, audio_max, video_min, video_max, screen_min, screen_max,
+      data_min, data_max;
+  GlobalConfiguration::GetPortRanges(audio_min, audio_max, video_min, video_max,
```

Comment:
Please describe in API reference that IceUnbundle will be enabled if port range is set to some specific values.

Reply:
done